suite-native/transactions/src/components/TransactionsList/TransactionListItem.tsx

```diff
@@ -76,7 +81,7 @@ export const TransactionListItem = memo(({ transaction }: AccountTransactionList
                     <Icon name={transactionIconMap[transaction.type] ?? 'placeholder'} />
                 </Box>
                 <Box>
-                    <Text>{transaction.type}</Text>
+                    <Text>{transactionTextMap[transaction.type]}</Text>
```

Comment:
What will happen if transaction type won't be Received or Sent? I am just wondering why TS doesn't complain here. Please add some fallback to just display directly `transaction.type` or something.